src/applications/facility-locator/containers/VAMap.jsx

```diff
@@ -500,14 +500,13 @@ class VAMap extends Component {
                   query={this.props.currentQuery}
                 />
               </div>
-              {results &&
-                results.length > 0 && (
-                  <Pagination
-                    onPageSelect={this.handlePageSelect}
-                    page={currentPage}
-                    pages={totalPages}
-                  />
-                )}
+              <PaginationWrapper
+                handlePageSelect={this.handlePageSelect}
+                currentPage={currentPage}
+                totalPages={totalPages}
+                results={results}
+                inProgress={this.props.currentQuery.inProgress}
```

Comment:
can we use the value from the constant above?
   ` const {
      currentQuery,
.....
    } = this.props;`

`inProgress={currentQuery.inProgress}`